contracts/DelFiPrice.sol

```diff
@@ -22,33 +22,16 @@ contract DelFiPrice is OpenOracleView {
      * @param messages The messages to post to the oracle
      * @param signatures The signatures for the corresponding messages
      */
-    function postPrices(bytes[] calldata messages, bytes[] calldata signatures) external {
+    function postPrices(bytes[] calldata messages, bytes[] calldata signatures, string[] calldata symbols) external {
```

Comment:
I hesitate to ask once again, but with the storage layout improvements, did we still think that it was important to have this to save gas, or was it that building the set is just kind of gross?

Reply:
It was costly

Reply:
I *think* it's actually so you can take in a list of 10 reported prices but only pay to post 3 of them.

Reply:
to only post 3 of them it would be better to only post the 3 you like ( since they are individually signed )

Reply:
I feel like we should be ok making the post signature more complex to assist on-chain. I think both are nice features: to be able to just put data without recalculating median sometimes, or recalculate without putting data, and also to save some gas / code surface area (there was at least one bug in the prior impl). Not sure what the real downside is?

Reply:
less solidity I think wins the day, but in general it's just kind of a code smell to me that we would ever need it in the first place, I suppose in the end it's just indicating the limitations of solidity, but wanted to make sure it wasn't indicating an issue in what we were doing.

Reply:
like, put prices + poke median would probably be the more clear design if we didn't have the assumption that we want to poke median almost every transaction that puts prices

Reply:
Yeah I think 2 functions would be less conflated but probably less convenient too, that said I don't think I have the same aversion to the 3rd arg